baselines/logging/test/gapic_logging_service_v2_v2.ts.baseline

```diff
@@ -749,16 +741,12 @@ describe('v2.LoggingServiceV2Client', () => {
             const request = generateSampleMessage(new protos.google.logging.v2.ListLogsRequest());
             request.parent = '';
             const expectedHeaderRequestParams = "parent=";
-            const expectedResponse = [
-              generateSampleMessage(new protosTYPE_STRING()),
-              generateSampleMessage(new protosTYPE_STRING()),
-              generateSampleMessage(new protosTYPE_STRING()),
-            ];
+            const expectedResponse = [new String(), new String(), new String()];
             client.descriptors.page.listLogs.createStream = stubPageStreamingCall(expectedResponse);
             const stream = client.listLogsStream(request);
             const promise = new Promise((resolve, reject) => {
-                const responses: protosTYPE_STRING[] = [];
-                stream.on('data', (response: protosTYPE_STRING) => {
+                const responses: protos.google.protobuf.FieldDescriptorProto.Type.TYPE_STRING[] = [];
```

Comment:
This does not look right - must be `string` here

Reply:
fixed it!